resources/public/api/conf/1.0/testdata/get-investor-payment.md

```diff
@@ -17,7 +17,7 @@
   "paymentStatus": "Paid",
   "paymentDate": "2017-06-20",
   "paymentAmount": 1000,
-  "paymentReference": "002630000993",
+  "paymentReference": "0000002630000993",
```

Comment:
Given this is a change in the test scenario, do we potentially need to add this to the next lot of comms going out? Otherwise is there a possibility that someone testing against external test will suddenly start seeing a failure for this case?